packages/libxml2.rb

```diff
@@ -29,23 +29,12 @@ class Libxml2 < Autotools
   def self.patch
     # Fix encoding.c:1961:31: error: ‘TRUE’ undeclared (first use in this function)
     system "for f in $(grep -rl 'TRUE)'); do sed -i 's,TRUE),true),g' $f; done"
-  end
-
-  configure_options '--enable-shared \
-      --enable-static \
-      --with-pic \
-      --without-python \
-      --without-lzma \
-      --with-zlib \
-      --with-icu \
-      --with-threads \
-      --with-history'
-
-  def self.check
     # Remove EBCDIC test since it fails.
     # Check https://mail.gnome.org/archives/xml/2010-April/msg00010.html for details.
-    system 'rm', 'test/ebcdic_566012.xml'
-
-    system 'make', 'check'
+    FileUtils.rm 'test/ebcdic_566012.xml'
```

Comment:
If we aren't even testing anymore, is there a need to remove a failing test?

Reply:
I used `run_tests` but building with tests led to the install including the tests too, which we don't want.

Reply:
Running some tests is better than running no tests at all because one test is failing the whole suite.  Totally in agreement with this decision.

Reply:
Yes, but since `run_tests` has been removed, we aren't running any tests at all. Hence, why would we modify the tests if we don't even run them?

Reply:
AFAIK, `run_tests` is a valid boolean property and still  works with `buildsystems/autotools` builds.

Reply:
I mean it has been removed from the libxml2 package file.

Reply:
Oh, was it ever there?

Reply:
There used to be an equivalent `self.check` section, which this PR removed while still leaving the test modification line behind.